src/standbypool/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure CLI Standbypool Extension #
+This is an extension to Azure CLI to manage Standbypool resources.
+
+## How to use ##
+Please add commands usage here.
```

Comment:
Please complete this readme file

Reply:
Thanks for pointing out, updated readme file.